requirements.txt

```diff
@@ -0,0 +1 @@
+six>=1.12.0
```

Comment:
if six is just used for unicode, we could have added just 

//ensure unicode function exists in python3 until python2 is deprecated.
try: unicode = str; except: pass

tho since you have started to use the better documented features, then i guess its ok, could you check if six is included in the ckan core for master 2.9.x etc?

Reply:
Yes, latest master has six==1.15.0